subt/script/check_rosK2.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+trap "exit;" HUP INT TERM
+trap "kill 0" EXIT
```

Comment:
There is no need for this here since nothing is run in the background. I the other scripts it is used to kill all background processes when the foreground process ends.

Reply:
ok